spec/controllers/static_page_controllers_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+RSpec.describe StaticPagesController do
```

Comment:
You could simplify this to be like

' RSpec.describe StaticPagesController do
  it "about page is successful" do
    get :about
    expect(response.status).to eq 200
  end

  it "contact page is successful" do
    get :contact
    expect(response.status).to eq 200
  end

  it "terms page is successful" do
    get :terms
    expect(response.status).to eq 200
  end
end'

because you don't actually need the individual describe blocks as you are only testing the one thing inside of it.